src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetSentencesForPersonServiceTest.kt

```diff
@@ -12,181 +13,180 @@ import org.springframework.boot.test.mock.mockito.MockBean
+            ),
+          ),
+        )
 
-        verify(probationOffenderSearchGateway, VerificationModeFactory.times(1)).getPerson(pncId = pncId)
+        val result = getSentencesForPersonService.execute(pncId)
+
+        result.shouldBe(Response(data = listOf(nomisSentence1, nomisSentence2)))
       }
 
-      it("returns errors when a person cannot be found by PNC ID in prisoner offender search") {
-        whenever(prisonerOffenderSearchGateway.getPersons(pncId = pncId)).thenReturn(
+      it("returns probation sentences only when the person cannot be found in Prisoner Offender Search") {
+        whenever(getPersonService.execute(pncId = pncId)).thenReturn(
           Response(
-            data = emptyList(),
-            errors = listOf(
-              UpstreamApiError(
-                causedBy = UpstreamApi.NOMIS,
-                type = UpstreamApiError.Type.ENTITY_NOT_FOUND,
-              ),
+            data = mapOf(
+              "prisonerOffenderSearch" to null,
+              "probationOffenderSearch" to personFromProbationOffenderSearch,
             ),
           ),
         )
 
-        val response = getSentencesForPersonService.execute(pncId)
+        val result = getSentencesForPersonService.execute(pncId)
 
-        response.errors.shouldHaveSize(1)
+        result.shouldBe(Response(data = listOf(nDeliusSentence1, nDeliusSentence2)))
       }
 
-      it("returns errors when a person cannot be found by a PNC ID in probation offender search") {
-        whenever(probationOffenderSearchGateway.getPerson(pncId = pncId)).thenReturn(
+      it("returns no sentences when the person cannot be found in either Prisoner Offender Search or Probation Offender Search") {
+        whenever(getPersonService.execute(pncId = pncId)).thenReturn(
           Response(
-            data = null,
-            errors = listOf(
-              UpstreamApiError(
-                causedBy = UpstreamApi.NDELIUS,
-                type = UpstreamApiError.Type.ENTITY_NOT_FOUND,
-              ),
+            data = mapOf(
+              "prisonerOffenderSearch" to null,
+              "probationOffenderSearch" to null,
             ),
           ),
         )
 
-        val response = getSentencesForPersonService.execute(pncId)
+        val result = getSentencesForPersonService.execute(pncId)
 
-        response.errors.shouldHaveSize(1)
+        result.shouldBe(Response(data = emptyList()))
       }
     }
 
-    describe("Nomis sentence search") {
+    describe("Nomis sentences") {
       it("retrieves bookind Ids for a person from Nomis using a Nomis number") {
         getSentencesForPersonService.execute(pncId)
 
-        verify(nomisGateway, VerificationModeFactory.times(1)).getBookingIdsForPerson(id = prisonerNumber)
+        verify(nomisGateway, VerificationModeFactory.times(1)).getBookingIdsForPerson(id = nomisNumber)
+      }
+
+      it("does not return sentences when booking IDs are not present") {
```

Comment:
```suggestion
      it("does not return prison sentences when booking IDs are not present") {
```

Even though it's within a NOMIS describe block, just be super clear?